docker-compose.local.yml

```diff
@@ -0,0 +1,142 @@
+      - './routes:/app/routes'
+      - './tests:/app/tests'
+      - './public:/app/public'
+      - './resources:/app/resources'
+      - 'volume.platform.test:/opt/data'
+      - './.local-deploy/accessibility-app/entrypoint.sh:/tmp/entrypoint.sh' # create different entrypoint file
+      - './.local-deploy/accessibility-app/etc/nginx/includes:/etc/nginx/includes' # add include for http & https blocks
+      - './.local-deploy/accessibility-app/etc/nginx/nginx.conf:/etc/nginx/nginx.conf' # replace primary nginx config
+      - './.local-deploy/accessibility-app/php.ini-development:/usr/local/etc/php/php.ini-development' # Deploy development php settings
+    networks:
+      - network.platform.mail
+      - network.platform.mysql
+      - network.platform.proxy
+      - network.platform.redis
+    depends_on:
+      platform.mysql:
+        condition: service_healthy
+      platform.redis:
+        condition: service_healthy
+      platform.meilisearch:
+        condition: service_healthy
+      platform.selenium:
+        condition: service_started
+    healthcheck:
+      test: ["CMD", "wget", "--no-verbose", "--spider",  "http://localhost:8080/nginx_status"]
+      retries: 3
+      timeout: 5s
+
+  platform.mysql:
+    container_name: platform.mysql
+    image: docker.io/bitnami/mysql:8.0
+    ports:
+      - '${FORWARD_DB_PORT:-3306}:3306'
+    environment:
+      MYSQL_DATABASE: '${DB_DATABASE}'
+      MYSQL_ROOT_PASSWORD: '${DB_PASSWORD}'
+    volumes:
+      - 'volume.platform.mysql:/bitnami/mysql/data'
+      - '.local-deploy/accessibility-db/opt/bitnami/mysql/conf/my_custom.cnf:/opt/bitnami/mysql/conf/my_custom.cnf'
+    networks:
+      - network.platform.mysql
+    healthcheck:
+      test: ["CMD", "mysqladmin", "ping", "-p${DB_PASSWORD}"]
+      retries: 3
+      timeout: 5s
+
+  platform.redis:
+    container_name: platform.redis
+    image: 'bitnami/redis'
+    ports:
+      - '${FORWARD_REDIS_PORT:-}:6379'
+    environment:
+      REDIS_HOST: '${REDIS_HOST}'
+      REDIS_PASSWORD: '${REDIS_PASSWORD}'
+      REDIS_PORT: '${REDIS_PORT}'
+    volumes:
+      - 'volume.platform.redis:/data'
+    networks:
+      - network.platform.redis
+    healthcheck:
+      test: ["CMD", "redis-cli", "ping"]
+      retries: 3
+      timeout: 5s
+
+  platform.meilisearch:
+    container_name: platform.meilisearch
+    image: 'getmeili/meilisearch:latest'
+    ports:
+      - '${FORWARD_MEILISEARCH_PORT:-7700}:7700'
+    volumes:
+      - 'volume.platform.meilisearch:/data.ms'
+    networks:
+      - network.platform.mail
+    healthcheck:
+      test: ["CMD", "wget", "--no-verbose", "--spider",  "http://localhost:7700/health"]
+      retries: 3
+      timeout: 5s
+
+  platform.mailhog:
+    container_name: platform.mailhog
+    image: 'mailhog/mailhog:latest'
+    ports:
+      - '${FORWARD_MAILHOG_PORT:-1025}:1025'
+      - '${FORWARD_MAILHOG_DASHBOARD_PORT:-8025}:8025'
+    networks:
+      - network.platform.mail
```

Comment:
We're using [mailpit](https://github.com/axllent/mailpit) now instead of mailhog because the latter isn't well supported anymore.

Reply:
Changed local deployment to use **mailpit** instead.